mlibrary/lib.umich.edu
# Updated iframe src and npm package

# Overview
The design system team is working on UI improvements to the Ask a Librarian chat widget. 

See the Jira ticket for more information:
> [WEBSITE-145 - (ON 7/30) Change skin id for embedded Ask a Librarian chat widget](https://mlit.atlassian.net/jira/software/projects/WEBSITE/boards/27?selectedIssue=WEBSITE-145)

There's also a tiny bit of padding on the bottom of the iframe on the [ask-librarian page](https://lib.umich.edu/ask-librarian). This update also removes that bit of extra padding.

**Note: This pull request should be merged on 7/30.**

## Testing

1. Visit the [ask-librarian page](https://deploy-preview-297--future-wwwlib-previews.netlify.app/) and look at the hard-coded widget to the right (if viewing full screen) or at the bottom (if on mobile).
2. Open the Ask a Librarian widget at the bottom of any page (LINK COMING AFTER PR BUILD SUCCEEDS).
3. Compare the chat widget UI with the [current, live version of the site](https://lib.umich.edu/ask-librarian).
4. Both instances of the widget should look like [the new widget UI](https://libraryh3lp.com/chat/umlibraryaskalibrarian@chat.libraryh3lp.com?skin=36020)

- Make sure the PR is consistent in these browsers:
  - [x] Safari (I do not have Safari since I'm Windows / Linux)
- Run accessibility tests:
  - [x] Lighthouse
